README.md

```diff
@@ -189,6 +189,54 @@ image = pipeline(prompt, num_inference_steps=50).images[0]
 
 Follow [this guide](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv) to know more. 
 
+
+### Experimental results through various scheduler settings:
+
+<details>
+<summary>
+We have converted fine-tuned checkpoint for the dog images into Diffusers compatible StableDiffusionPipeline and ran various experiments with different scheduler settings. For example, the following parameters of the `DDIMScheduler` are tested on a different set of `guidance_scale` and `num_inference_steps`.
+</summary>
```

Comment:
The code quotes are not rendering:

<img width="841" alt="image" src="https://user-images.githubusercontent.com/22957388/217406174-c782e0d7-a0c8-4523-948e-41e69b84ef88.png">